swagger.yml

```
@@ -108,6 +113,19 @@ definitions:
       - FERN_ABSATZ_GESCHAEFT
       - AUSSER_GESCHAEFTSRAUM_VERTRAG
 
+  kontoumsaetze:
+    type: object
+    properties:
+      umsatzdaten:
+        type: string
```

wenn das hier Base64 kodierte Daten sein sollen, ist das so eindeutiger

```suggestion
        type: string
        format: byte
```